indy_node/server/node.py

```diff
@@ -319,52 +318,8 @@ def authNr(self, req):
         else:
             return super().authNr(req)
 
-    def isSignatureVerificationNeeded(self, msg: Any):
```

Comment:
We need to remove it in plenum too, and make request handlers decide whether signature verification is needed.

Reply:
`isSignatureVerificationNeeded` is intended for general query txns, like GET_TXN and GET_TXNS, i will change `openTxns`